src/qml/FeatureListForm.qml

```diff
@@ -158,6 +158,7 @@ Rectangle {
         Text {
           anchors { horizontalCenter: parent.horizontalCenter; verticalCenter: parent.verticalCenter }
           font.bold: true
+          font.pointSize: Theme.resultFont.pointSize
```

Comment:
For the record: it could well be that we'll need to adjust this point size when looking at the result on a smartphone. We just didn't set a font size before, which left it some mysterious internal mechanics to decide was the size would be.